app/kubernetes-ingress-controller/1.0.x/concepts/ingress-versions.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Ingress v1 and v1beta1 differences
+---
+
+## Introduction
+
+Kubernetes 1.18 introduced a new v1 API for the [Ingress resource][kubernetes-ingress-doc].
+It standardizes common practices and clarifies implementation requirements that
+were previously up to individual controller vendors. This document covers those
+changes as they relate to Kong Ingress Controller and provides example
+equivalent v1beta1 and v1 resources for comparison.
+
+## Paths
+
+Both Ingress v1beta1 and v1 HTTP rules require a path, which represents a [URI
+path][uri-rfc-paths]. Although v1beta1 had specified that paths were [POSIX
+regular expressions][posix-regex] and enforced this, but in practice most
+controllers used other other implementations that did not match the
+specification. v1 seeks to reduce confusion by introducing several i[path
+types][path-types] and lifting restrictions on regular expression grammars used
+by controllers.
+
+### v1beta1
+
+The controller passes paths directly to Kong and relies on its [path handling
```

Comment:
```suggestion
For the `spec.rules[*].http.paths[*].path` field, the controller passes paths directly to Kong and relies on its [path handling
```

Reply:
There aren't any other paths in the Ingress spec that these could be confused with, so IMO this isn't adding much beyond making the sentence longer. Leaving this as-is.